README.md

```diff
@@ -1,6 +1,10 @@
 # Minio plugin
 
 [![Build Status](https://ci.jenkins.io/job/Plugins/job/minio-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/minio-plugin/job/master/)
+[![License](https://img.shields.io/github/license/jenkinsci/minio-plugin.svg?color=green)](https://github.com/jenkinsci/minio-plugin/blob/master/LICENSE)
```

Comment:
IIRC it is best practice to mention the license of a project within the reademe, however I am not 100% sure on that :thinking:

Reply:
It certainly can't hurt to add